src/app/news/news-list/news-list.component.html

```diff
@@ -9,16 +9,20 @@ <h4>Всі новини</h4>
       <div class="col">
          <div class="card-columns">
             <div *ngFor="let one of news" class="card mb-0 mb-sm-2 shadow">
-               <ul class="list-group list-group-flush">
+               <ul class="list-group list-group-flush" style="height: 46px">
```

Comment:
Adding inline styles is a bad practice. There is no way we'll have them in this project.
https://dev.to/alim1496/avoid-using-inline-css-styles-5b6p